### PR TITLE
Add blog post about ansible support

### PR DESCRIPTION
**Pull request by gciavarrini**
This PR must me merged **AFTER** 
https://github.com/project-flotta/flotta-operator/pull/216
https://github.com/project-flotta/flotta-device-worker/pull/241

Eventually, this PR should be updated to reflect changes during the operator and device PRs review.

### CODE REVIEW

File: _posts/2022-09-28-run-ansible-playbooks.markdown [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- flotta
+- devices
+- ansible
+
+summary-1:
+---
+There may be cases n which you would like to be able to execute a scripts or commands in a device or on a group of devices.
+_Project Flotta_ makes your life easier by supporting _Ansible_ playbook execution.
+How can we create an _Ansible_ playbook for the edge devices? How does the execution work in _Project Flotta_?
+his is what we will cover in this blog post.
```
Comment (pkliczewski) [inline _posts/2022-09-28-run-ansible-playbooks.markdown:17]: s/his/This

File: _posts/2022-09-28-run-ansible-playbooks.markdown [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+categories: flotta
+author: Gloria Ciavarrini <gciavarrini@redhat.com>
+tags:
+- flotta
+- devices
+- ansible
+
+summary-1:
+---
+There may be cases n which you would like to be able to execute a scripts or commands in a device or on a group of devices.
```
Comment (pkliczewski) [inline _posts/2022-09-28-run-ansible-playbooks.markdown:14]: I would add that during life cycle of the device we may need to change configuration without rebooting a device which is needed by rpm ostree.

File: _posts/2022-09-28-run-ansible-playbooks.markdown [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+# Architecture
+What happens when _Flotta Operator_ receives the `EdgeConfig`?
+
+It automatically creates a `PlaybookExecution` CR for each device that has been properly labelled.
+
+<img src="/assets/images/architecture_ansible_support.png" alt="High level architecture of Ansible support" width="1100"/>
+
+In this way it is possible to monitor the execution of the playbook on each edge device, indeed the edge device will update the status of the `PlaybookExecution` CR according with the Ansible playbook execution result.
```
Comment (pkliczewski) [inline _posts/2022-09-28-run-ansible-playbooks.markdown:91]: I would add what happens when a device is rebooted while playbook is being run and how many times plabook will be run on a device.

**Review by pkliczewski — COMMENTED** [review]
Overall looks good, few comments

File: _posts/2022-09-28-run-ansible-playbooks.markdown (hunk 2 above)
Comment (gciavarrini) [reply _posts/2022-09-28-run-ansible-playbooks.markdown]: Added a sentence. WDYT?

File: _posts/2022-09-28-run-ansible-playbooks.markdown (hunk 3 above)
Comment (gciavarrini) [reply _posts/2022-09-28-run-ansible-playbooks.markdown:93]: Thanks for the suggestions!
> how many times plabook will be run on a device

Added a paragraph to describe the possible execution strategies

> when a device is rebooted while playbook is being run 

I'd like to wait for reviews in the device-worker PR so I can write a "matching" description of the rebooting case.
Then I'll ask you to re-review if you don't mind :)

**Pull request merged**